src/components/Card.tsx

```diff
@@ -8,10 +8,10 @@ export function Card(props: { card: ICard }) {
     return (
         <Link
             href={`/cards/${id}`}
-            className="w-80 h-52 overflow-hidden rounded-3xl border-2 -mt-36 first:mt-0"
+            className="w-80 h-52 overflow-hidden rounded-3xl border-2 -mb-36 hover:-mb-12 transition-all"
         >
             <img
```

Comment:
Let's change the img into Image. Created [a ticket](https://tsoibet.atlassian.net/browse/KAADO-54) to address this.